backend/repositories/GameRepository.ts

```diff
@@ -0,0 +1,25 @@
+import { GameData } from '../types/Game'
+import { InputError } from '../errors/InputError'
+import * as storage from '../storage/Storage'
+
+const isValidGame = (gameData: GameData): boolean => {
+  if (!gameData.name || !gameData.description) {
```

Comment:
```
if (X) {
  return false
} else if (Y) {
  return false
}
```

could be further improved as 
```
if (X || Y) {
  return false
}
```

Reply:
fixed in https://github.com/salsita/foosball-rating/pull/113/commits/d8bb46a15e9ce4e8733aba65b651d0ec0eece570